ios/RNZcash.swift

```diff
@@ -440,7 +441,17 @@ class RNZcash: RCTEventEmitter {
 
   // Events
   public func sendToJs(name: String, data: Any) {
-    self.sendEvent(withName: name, body: data)
+    if (hasListeners) {
+      self.sendEvent(withName: name, body: data)
+    }
+  }
+
+  override func startObserving() -> Void {
+    hasListeners = true
+  }
+
+  override func stopObserving() -> Void {
```

Comment:
Where are `startObserving` and `stopObserving` called?

Reply:
These override methods in the React Native base class. React Native calls them to let us know when it's ready or not.